<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_new</class>
 <widget class="QDialog" name="dialog_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid_layout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vertical_layout">
      <item>
       <widget class="QLineEdit" name="text_name">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="box_version">
        <property name="placeholderText">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_fabric">
        <property name="text">
         <string>Use Fabric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_java">
        <property name="placeholderText">
         <string>Java Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_jvm">
        <property name="placeholderText">
         <string>JVM Arguments</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontal_layout">
        <item>
         <widget class="QPushButton" name="button_accept">
          <property name="text">
           <string>Accept</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
